src/Microsoft.DotNet.Build.CloudTestTasks/UploadClient.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Globalization;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.DotNet.Build.CloudTestTasks
+{
+    using Task = System.Threading.Tasks.Task;
```

Comment:
outside (I'd probably specify the TaskLoggingHelper myself, since that type is used fewer times in this file, but that's just a preference)